.github/workflows/build.yaml

```diff
@@ -23,4 +23,4 @@ jobs:
           restore-keys: |
             mkdocs-material-
       - run: pip install -r requirements.txt
-      - run: mkdocs gh-deploy --force
+      - run: mkdocs gh-deploy --force # TODO: add -s (strict) switch
```

Comment:
Last action before merge - turn on strict builds when all other changes land.

Reply:
Would be good if this can be configured to run `mkdocs build -s` on PRs as well as pushes to main. Would involve separating it from the gh-deploy i guess, but it would catch issues when you update versions of components in this repo before you merge it.

Reply:
That makes sense to me.